src/Components/Shared/Modal/index.jsx

```diff
@@ -3,6 +3,8 @@ import styles from './modal.module.css';
 import Button from '../Button';
 
 const SharedModal = (data) => {
+  let containerStyle = styles.content;
```

Comment:
We should declare the variable but its not necessary assign a content

Reply:
Hi Fede, this is just a query to better understand your logic. Why do you use let and not const? Do you need it to change its value in some instance?

Reply:
Because the containerStyle will not change outside the switch, it is not constant (and because eslint doesn't let me use const).